core/types/calls.cc

```diff
@@ -1094,6 +1095,102 @@ class T_revealType : public IntrinsicMethod {
     }
 } T_revealType;
 
+class T_LazyConstants_lazyIsA_p : public IntrinsicMethod {
+public:
+    void apply(const GlobalState &gs, DispatchArgs args, const Type *thisType, DispatchResult &res) const override {
+        constexpr string_view method = "T::LazyConstants.lazy_is_a?";
+
+        if (args.args.size() != 2) {
+            return;
+        }
+
+        // TODO(jez) Might want to point to the argument to the call, not the call itself, if we have that info.
+        auto loc = Loc(args.locs.file, args.locs.call);
```

Comment:
I think that this is `args.locs.args[1]` in this case?

Reply:
Perfect, thanks! I don't know how I didn't find that.